example_14/hardware/rrbot_actuator_without_feedback.cpp

```diff
@@ -183,21 +183,23 @@ hardware_interface::return_type RRBotActuatorWithoutFeedback::read(
 hardware_interface::return_type ros2_control_demo_example_14::RRBotActuatorWithoutFeedback::write(
   const rclcpp::Time & /*time*/, const rclcpp::Duration & /*period*/)
 {
-  // START: This part here is for exemplary purposes - Please do not copy to your production code
-  RCLCPP_INFO(
-    rclcpp::get_logger("RRBotActuatorWithoutFeedback"), "Writing command: %f", hw_joint_command_);
-
-  // Simulate sending commands to the hardware
-  std::ostringstream data;
-  data << hw_joint_command_;
-  RCLCPP_INFO(
-    rclcpp::get_logger("RRBotActuatorWithoutFeedback"), "Sending data command: %s",
-    data.str().c_str());
-  send(sock_, data.str().c_str(), strlen(data.str().c_str()), 0);
+  if (!std::isnan(hw_joint_command_))
```

Comment:
```suggestion
  if (std::isfinite(hw_joint_command_))
```

May be `isfinite` is better here as it handles both `NaN` and `Infinity`